### PR TITLE
LRDOCS-7533 Port Creating Pages section to Site Building

### PR DESCRIPTION
**Pull request by mwilliams2014**

### CODE REVIEW

**Comment by mwilliams2014** [comment]
This includes 7.3 information for these tickets:
https://issues.liferay.com/browse/LPS-104520
https://issues.liferay.com/browse/LPS-102734
https://issues.liferay.com/browse/LPS-103634

File: docs/dxp/7.2.x/en/site-building/02-creating-pages/03-content-page-elements.md [hunk 1]
```diff
@@ -46,11 +47,27 @@ The Widgets section functions just like the *Add* menu on a Widget Page. The ful
...
 
+## Contents
+
+Since Liferay DXP 7.3, the Contents section provides a list of the web content that's used on the page. This includes content displayed in a widget and content mapped to content fields. Click the *Contents* button (![Contents](../../images/icon-contents.png)) to open the Contents panel. From this section, you can perform these actions for each piece of content used on the page:
+
+- *Edit:* Edit the web content 
+- *Permissions:* Update the web content's permissions
+- *View Usages:* See where the web content is used on the site
+
+>**Note:** To edit inline content, you must have the "Edit Inline Content" permission assigned to your Role. However, if you don't have permission to edit the page, you can still edit content (web content, blog, etc.) that you have permission to edit that's displayed through one of these methods: mapped content, web content display, or an Asset Publisher (with manual selection).
```
Comment (ruben-pulido) [inline docs/dxp/7.2.x/en/site-building/02-creating-pages/03-content-page-elements.md]: 👍  Looks good to me!

**Comment by jrhoun** [comment]
Just started reviewing.

**Comment by jrhoun** [comment]
Closing this out as I have merged most of the articles in - though as we discussed @mwilliams2014 - I only have fully reviewed 3 - those three are visible and uncommented. The rest I'll continue to review.

**Pull request closed without merging**